### PR TITLE
HEAL preprod 1615839591

### PR DESCRIPTION
**Pull request by themarcelor**
copying files from qa-heal.planx-pla.net to preprod.healdata.org

### CODE REVIEW

File: preprod.healdata.org/portal/gitops.json [hunk 1]
```diff
@@ -6,21 +6,28 @@
     "projectDetails": "boardCounts"
   },
   "components": {
-    "appName": "Generic Data Commons Portal",
+    "appName": "HEAL QA Data Commons Portal",
```
Comment (xritter1) [inline preprod.healdata.org/portal/gitops.json:9]: "QA" should be "Preprod"

File: preprod.healdata.org/portal/gitops.json [hunk 2]
```diff
@@ -6,21 +6,28 @@
...
     "index": {
       "introduction": {
         "heading": "Data Commons",
-        "text": "The HEAL preprod Data Commons supports the management, analysis and sharing of data for the research community.",
+        "text": "The HEAL QA Data Commons supports the management, analysis and sharing of data for the research community.",
```
Comment (xritter1) [inline preprod.healdata.org/portal/gitops.json:13]: "QA" should be "Preprod"

File: preprod.healdata.org/portal/gitops.json [hunk 3]
```diff
@@ -6,21 +6,28 @@
...
         "link": "/submission"
       },
       "buttons": [
         {
           "name": "Define Data Field",
           "icon": "data-field-define",
-          "body": "The HEAL preprod Data Commons define the data in a general way. Please study the dictionary before you start browsing.",
+          "body": "The HEAL QA Data Commons define the data in a general way. Please study the dictionary before you start browsing.",
```
Comment (xritter1) [inline preprod.healdata.org/portal/gitops.json:20]: "QA" should be "Preprod"

File: preprod.healdata.org/portal/gitops.json [hunk 4]
```diff
@@ -6,21 +6,28 @@
...
           "link": "/DD",
           "label": "Learn more"
         },
+        {
+          "name": "Explore Data",
+          "icon": "data-explore",
+          "body": "The Exploration Page gives you insights and a clear overview under selected factors.",
+          "link": "/explorer",
+          "label": "Explore data"
+        },
```
Comment (xritter1) [inline preprod.healdata.org/portal/gitops.json:30]: Exploration Page should not be in preprod afaik

File: preprod.healdata.org/portal/gitops.json [hunk 5]
```diff
@@ -38,18 +45,18 @@
       ]
     },
     "navigation": {
-      "title": "HEAL preprod Data Commons",
+      "title": "HEAL QA Data Commons",
```
Comment (xritter1) [inline preprod.healdata.org/portal/gitops.json:48]: "QA" should be "Preprod"

Comment (williamhaley) [reply preprod.healdata.org/portal/gitops.json:48]: Thanks @xritter1!

File: preprod.healdata.org/manifests/hatchery/hatchery.json [hunk 6]
```diff
@@ -0,0 +1,129 @@
+{
+  "user-namespace": "jupyter-pods-qa-heal",
+  "sub-dir": "/lw-workspace",
+  "user-volume-size": "10Gi",
+  "sidecar": {
+    "cpu-limit": "1.0",
+    "memory-limit": "256Mi",
+    "image": "quay.io/cdis/gen3fuse-sidecar:2021.03",
+    "env": {
+      "NAMESPACE": "qa-heal",
```
Comment (williamhaley) [inline preprod.healdata.org/manifests/hatchery/hatchery.json:10]: @xritter1 I'm assuming I should find/replace just about everywhere in the automated changes and replace `qa` with `preprod`?

**Comment by PlanXCyborg** [comment]
# preprod.healdata.org/manifest.json
## Deployment changes
- fence
  - REQUIRES A FENCE DB MIGRATION, since with this update there is now an additional column in the authorization_code table "refresh_token_expires_in" (uc-cdis/uc-cdis/fence#848)
- ssjdispatcher
  - `metadata-service` must already be present or added to the `"versions": {` section of the `cdis-manifest`. After `metadata-service` has been confirmed to be in the `cdis-manifest`, the now-required Metadata Service creds for SSJDispatcher must be configured by running `gen3 roll all` with `JENKINS_HOME` **not** set. `gen3 roll all` calls `gen3 kube-setup-ssjdispatcher`, which now [automatically configures](https://github.com/uc-cdis/cloud-automation/pull/1428) the SSJDispatcher secret with Metadata Service creds. (uc-cdis/uc-cdis/ssjdispatcher#32)

**Comment by williamhaley** [comment]
Lots of odd Jenkins/CI issues and some functional changes to the configs need to be made. Closing this out.

**Pull request closed without merging**